src/sdltiles.cpp

```diff
@@ -545,7 +545,9 @@ void refresh_display()
 #endif
 #if defined(__ANDROID__)
     draw_terminal_size_preview();
-    draw_quick_shortcuts();
+    if( g ) {
+        draw_quick_shortcuts();
```

Comment:
Isn't this hiding shortcuts in main menu (and other menus that are shown before game is started)?

Reply:
Three buttons "unknown key XX" will appear at the bottom (in the screenshot) and pressing them crashes the app.